tests/tests/test-xcm/test-xcmv3-new-instructions.ts

```diff
@@ -0,0 +1,382 @@
+import "@moonbeam-network/api-augment";
+import { expect } from "chai";
+import { CHARLETH_ADDRESS } from "../../util/accounts";
+import { describeDevMoonbeam } from "../../util/setup-dev-tests";
+import {
+  XcmFragment,
+  injectHrmpMessage,
+  RawXcmMessage,
+  sovereignAccountOfSibling,
+} from "../../util/xcm";
+import { expectEVMResult } from "../../util/eth-transactions";
+import { ALITH_TRANSACTION_TEMPLATE, createTransaction } from "../../util/transactions";
+
+// Here we are testing each allowed instruction to be executed. Even if some of them throw an error,
+// the important thing (and what we are testing) is that they are
+// executed and are not blocked with 'WeightNotComputable' due to using max weight.
+describeDevMoonbeam(
+  "XCM V3 Instructions",
+  (context) => {
+    let dotAsset: any;
+    let amount: bigint;
+    let paraId: number;
+
+    before("Set up initial constants", async function () {
+      paraId = 888;
+      const paraSovereign = sovereignAccountOfSibling(context, paraId);
+      const metadata = await context.polkadotApi.rpc.state.getMetadata();
+      const balancesPalletIndex = (metadata.asLatest.toHuman().pallets as Array<any>).find(
+        (pallet) => pallet.name === "Balances"
```

Comment:
same comment as in the `tests/tests/test-xcm/test-xcmv3-max-weight-instructions.ts`